composer.json

```diff
@@ -28,7 +28,7 @@
         "bin-dir": "bin"
     },
     "support": {
-        "irc": "irc://irc.freenode.org/forkcms",
+        "slack": "https://fork-cms.herokuapp.com",
```

Comment:
I don't think "slack" is a valid key in the "support" part?

Have you done "composer validate" after this change?